x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -454,35 +454,35 @@ process where wildcard(process_path, "*\\red_ttp\\wininit.*", "*\\abc\\*", "*def
 
 
 addOperator
-process where serial_event_id + 2 == 41
+process where serial_event_id + 2 == -2147483647
```

Comment:
They were changed so that the arithmetics optimisation won't kick in and so that the scripted translation can still be verified by the test (the new values would cause an overflow if optimised).